installing/install_config/customizations.adoc

```diff
@@ -23,7 +23,7 @@ the `oc explain` command, for example `oc explain builds --api-version=config.op
 All cluster configuration resources are globally scoped (not namespaced) and named `cluster`.
 ////
 Config changes should not require coordinated changes between config resources, if you find
-yourself struggling to update these docs to explain coordinated changes, please reach out 
+yourself struggling to update these docs to explain coordinated changes, please reach out
```

Comment:
I know it's not in the scope of this PR, but this is a run-on sentence that could use a period:
"Config changes should not require coordinated changes between config resources. If you find
yourself struggling..."

Reply:
It's also in a comment that will never be uncommented. :)